.storybook/__snapshots__/Storyshots.test.js.snap

```diff
@@ -20277,7 +20277,7 @@ exports[`Storyshots Textarea validation 1`] = `
   <textarea
     aria-describedby="error-asInput1 description-asInput1"
     aria-invalid={false}
-    className="form-control"
+    className="form-control "
```

Comment:
This is why the extra space is here

![image](https://user-images.githubusercontent.com/8136030/38994909-aab0cdbc-43b5-11e8-8bd8-9eaeef5f436a.png)

- [RunKit](https://runkit.com/jaebradley/5ad8991ff4c25d0012f5b2c3)

will this cause any problems?

Reply:
That's annoying but it won't cause any issues.